src/Microsoft.Framework.PropertyActivator.Internal/project.json

```diff
@@ -0,0 +1,15 @@
+{
+    "version": "1.0.0-*",
+    "shared": "*.cs",
+    "dependencies": {
```

Comment:
curious: similarly how does this one get away without a dependency on Microsoft.Framework.PropertyHelper.Internal?